.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+    "recommendations": [
+        "svelte.svelte-vscode",
+        "bradlc.vscode-tailwindcss",
+        "rust-lang.rust-analyzer",
+        "dbaeumer.vscode-eslint",
+        "esbenp.prettier-vscode"
+    ]
```

Comment:
It would be good to see a plugin that can run vite tests. I have tried 
zixuanchen.vitest-explorer, but it did nothing.

Reply:
Yes, the vitest plugins sadly don't work and I can't really find any configuration option that can save the situation.

Reply:
Nope, you are right I see it now, it works if you open only the client part. I will add `ZixuanChen.vitest-explorer` to the client extension recommendation.